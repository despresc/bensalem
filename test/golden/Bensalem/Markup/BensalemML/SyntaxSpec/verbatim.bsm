Verbatims\``|fuse with neighbours |``/ like this
\&foo \``|indentation
  works as well|``/

\testinl
