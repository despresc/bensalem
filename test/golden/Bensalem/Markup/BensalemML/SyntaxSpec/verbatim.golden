fromList
 [ PlainText "Verbatimsfuse"
 , LineSpace 1
 , PlainText "with"
 , LineSpace 1
 , PlainText "neighbours"
 , LineSpace 2
 , PlainText "like"
 , LineSpace 1
 , PlainText "this"
 , LineEnd
 , ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 49
     , srcLine = 2
     , srcCol = 1
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 54
     , srcLine = 2
     , srcCol = 6
     }
    }
   , elementName = "foo"
   , elementAttrs = NoAttrs
   , elementScopeContent = LayoutScopeContent
    ( fromList
     [ LineSpace 1
     , PlainText "indentation"
     , LineEnd
     , LineSpace 2
     , PlainText "works"
     , LineSpace 1
     , PlainText "as"
     , LineSpace 1
     , PlainText "well"
     ]
    )
   }
  )
 , LineEnd
 , LineEnd
 , ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 92
     , srcLine = 5
     , srcCol = 1
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 100
     , srcLine = 5
     , srcCol = 9
     }
    }
   , elementName = "testinl"
   , elementAttrs = NoAttrs
   , elementScopeContent = InlineScopeContent
   }
  )
 , LineEnd
 ]